### ISSUE TITLE
Tutorial outpost weapon holder slot is very small

### ISSUE THREAD

**Issue by Rokvach** [Regalis11/Barotrauma#9296]
**Description**
![image.png](https://images.zenhubusercontent.com/5c924117534a9b0a86cdddce/c95c35c1-88a3-42c8-aadf-03ca7885affe)

**Version**
Branch: bugfixes

**Comment by Regalis11** [comment]
This was actually more severe than it seemed.

Happened as a result of https://github.com/Regalis11/Barotrauma-development/commit/b3bb8ecd5466b99f373f6832dac9f740082a5eac in which I added support for upgrading the Containable restrictions of an ItemContainer to keep items with the tag "weapon" still placeable in the weapon holders in older subs. The problem was that due to the changes, the Inventory instance was recreated, but the UI wasn't, which prevented the new Inventory instance from figuring out which UI it belongs to and how it's supposed to render itself. I could imagine that leading to more severe side-effects than just the messed up inventory layout. So I implemented a less flexible but simpler fix for this in https://github.com/Regalis11/Barotrauma-development/commit/ae43e8c

> Now we just override the ContainableItems instead of recreating the whole inventory and the ContainableItems
